### PR TITLE
Tags optional

### PR DESCRIPTION
**Pull request by orthagonal**
https://github.com/firstandthird/logr/issues/5

### CODE REVIEW

File: test/logr.test.js [hunk 1]
```diff
@@ -144,6 +144,20 @@ describe('logr', () => {
       log(['tag1'], 'test');
       expect(lastMessage).to.equal('[default,tag1] test');
     });
+    it('should let you optionally leave out the tags ', () => {
```
Comment (jgallen23) [inline test/logr.test.js]: can you do a test without a defaultTag?

**Comment by orthagonal** [comment]
@jgallen23 updated to add this test case, also it will leave out the '[]' if no tags are declared.

**Comment by jgallen23** [comment]
looks good. merge master into this so I can accept PR

**Comment by orthagonal** [comment]
@jgallen23 can be merged

**Pull request merged**